qemu/tests/cfg/tpm_with_bitlocker.cfg

```diff
@@ -0,0 +1,20 @@
+- tpm_with_bitlocker:
+    only Windows
+    virt_test_type = qemu
+    type = tpm_with_bitlocker
+    required_qemu= [4.2.0,)
+    start_vm = yes
+    kill_vm = yes
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    tpms = tpm0
+    tpm_backend_tpm0 = emulator
+    tpm_model_tpm0 = tpm-crb
+    tpm_version_tpm0 = 2.0
+    Win2019, Win2008, Win2016, Win2012:
+        cmd_install_bitlocker = 'powershell -command "Install-WindowsFeature -Name BitLocker -Restart"'
```

Comment:
We could remove -Restart here, with this parameter here, I got error:
04:21:33 ERROR| ERROR 1-Host_RHEL.m8.u3.product_av.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.tpm_with_bitlocker.tpm_with_bitlocker.q35 -> ShellStatusError: Could not get exit status of command: 'powershell -command "Install-WindowsFeature -Name BitLocker -Restart"'    (output: '\nSuccess Restart Needed Exit Code      Feature Result                               \n------- -------------- ---------      --------------                               \nTrue    Yes            SuccessRest... {BitLocker Drive Encryption, Enhanced Stor...\nWARNING: You must restart this server to finish the installation process.\n\n\n\n')
04:21:33 INFO | 

And we have reboot vm operation just after bitlocker installation in .py already. 
        session.cmd(cmd_install_bitlocker, 360)
        session = vm.reboot(session, timeout=480)

Reply:
Thanks, have updated.